### PR TITLE
Use `setup-rails` for CI?

### PR DESCRIPTION
**Pull request by andyw8**
:wave:

I recently published [setup-rails](https://github.com/andyw8/setup-rails), a reusable GitHub CI workflow:

https://www.andywaite.com/2022/04/15/reusable-github-actions-rails-workflow.html

I'm hoping this would be a useful addition to Jumpstart. Here's an example of it in use:

http://github.com/andyw8/jumpstart-setup-rails

(I had to make a [small adjustment](https://github.com/andyw8/jumpstart-setup-rails/commit/2985743f4957e1a75735ee216a936aa9c95d81ba) to the fixtures).

I understand that not everyone wants to use GitHub Actions, but the file would have no effect if running on another CI platforms.

### CODE REVIEW

File: template.rb [hunk 1]
```diff
@@ -210,6 +210,20 @@ def add_esbuild_script
...
+  
+    name: Verify
+    on: [push]
+
+    jobs:
+      verify:
+        uses: andyw8/setup-rails/.github/workflows/verify.yml@v1
+  EOS
+  create_file '.github/workflows/verify.yml', script
+end
```
Comment (excid3) [inline template.rb:225]: Could we use `copy_file` instead for this? 

Might conflict with the repo's .github folder, so we might want to put a `github/workflows/verify.yml` instead and specify the location: https://rubydoc.info/gems/thor/0.15.4/Thor%2FActions:copy_file

**Comment by excid3** [comment]
This is great @andyw8. Meant to get to this way sooner, but better late than never!

**Comment by excid3** [comment]
RE: the fixtures changes, should we override the fixtures file? Right now we're just using the fixtures file that Devise generates.

**Comment by andyw8** [comment]
@excid3 would you be interested in using this on Jumpstart Pro?

**Pull request merged**